li-utils/src/main/pegasus/com/linkedin/common/TermSource.pdl

```diff
@@ -0,0 +1,18 @@
+namespace com.linkedin.common
+
+/**
+ * TermSource
+ */
+enum TermSource {
```

Comment:
Suggest we remove this enum and move back to string if needed, enums are very inflexible. 
See my comment on using (and extending) ExternalReference model.

Reply:
will revert back the enum changes